src/cloudevents.ts

```diff
@@ -15,6 +15,19 @@
 import * as express from 'express';
 import {CloudEventsContext} from './functions';
 
+/**
+ * Custom exception class to represent errors durring event conversions.
+ */
+export class EventConversionError extends Error {}
+
+// CloudEvent service names.
+export const FIREBASE_AUTH_CE_SERVICE = 'firebaseauth.googleapis.com';
+export const FIREBASE_CE_SERVICE = 'firebase.googleapis.com';
+export const FIREBASE_DB_CE_SERVICE = 'firebasedatabase.googleapis.com';
+export const FIRESTORE_CE_SERVICE = 'firestore.googleapis.com';
+export const PUBSUB_CE_SERVICE = 'pubsub.googleapis.com';
+export const STORAGE_CE_SERVICE = 'storage.googleapis.com';
```

Comment:
I recommend one export for related values.

```ts
export const CE_SERVICE = {
  FIREBASE_AUTH: 'firebaseauth.googleapis.com',
  FIREBASE: 'firebase.googleapis.com',
  FIREBASE_DB: 'firebasedatabase.googleapis.com',
  FIRESTORE: 'firestore.googleapis.com',
  PUBSUB: 'pubsub.googleapis.com',
  STORAGE: 'storage.googleapis.com',
};
```

(Reverse NS for the identifiers) Then importing will be easier.